Tasks/PublishSymbols/SymbolsCommon.psm1

```diff
@@ -85,7 +85,18 @@ function Get-SymbolBinaryPath
         [string]$V1AgentPath
     )
 
-    $useDbgLkg = [System.Convert]::ToBoolean($env:PublishSymbols_UseDbgLkg)
+    $useDbgLkg = $env:PublishSymbols_UseDbgLkg
```

Comment:
>$env:PublishSymbols_UseDbgLkg [](start = 17, length = 29)

IS using Get-VstsTaskVariable better here ?

Reply:
I don't think I can use this because, as we were looking at together, this variable isn't among those read by the VstsTaskSdk by this line: foreach ($variable in (Get-ChildItem -Path Env:ENDPOINT_?*, Env:INPUT_?*, Env:SECRET_?*, Env:SECUREFILE_?*))

Reply:
The agent source suggests this comes in as VSTS_PUBLIC_VARIABLES per https://github.com/Microsoft/vsts-agent/blob/master/src/Agent.Worker/Handlers/Handler.cs
but it works as-is so I don't see the benefit of introducing abstraction for this simple case